web-page/src/components/filter-box/index.js

```diff
@@ -0,0 +1,215 @@
+import React, { useState } from 'react'
+import { connect } from 'react-redux'
+import { searchCards, addFilter, removeFilter } from '../../redux/modules/search'
+import './index.css'
+import Filter from '../Filter'
+import { parseFilterText, findArrayItemsInArrayOrString, getFieldsToSearch, mapIdToColor, createAllButtons } from './filter.utils'
```

Comment:
Split this into multiple lines.
```suggestion
import {
    parseFilterText,
    findArrayItemsInArrayOrString,
    getFieldsToSearch,
    mapIdToColor,
    createAllButtons
} from './filter.utils'
```

Reply:
Done!